aequilibrae/project/network/osm_downloader.py

```diff
@@ -9,12 +9,13 @@
 
 For the original work, please see https://github.com/gboeing/osmnx
 """
+import logging
 import time
 import re
 import requests
 from .osm_utils.osm_params import http_headers, memory
 from aequilibrae.parameters import Parameters
-from aequilibrae import logger
+from aequilibrae.context import get_logger
```

Comment:
again - i don't think that get_logger is used here.

Reply:
it's used on line 39: `self.logger = logger or get_logger()`